pkg/gadgets/traceloop/tracer/bpf/traceloop.bpf.c

```diff
@@ -73,6 +76,15 @@ struct {
 	__uint(max_entries, 512);
 } syscalls SEC(".maps");
 
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
+	__uint(key_size, sizeof(u64));
+	__uint(value_size, sizeof(bool));
```

Comment:
We don't care about the value, is there something in c/ebpf like `struct{}` in go?